index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/8b3d2372ca.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;0,700;1,400&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+
+    <div class="container">
+        <div class="navbar">
+            <div class="logo">
+                G
+            </div>
+            <div class="menu">
+                <i class="fas fa-bars"></i>
+            </div>
+        </div>
+        <div class="section">
+            <div class="hero">
+                <p>Hey! <br>
+                    Grace Here <br>
+                    Happy to see you!</p>
+            </div>
+            <div class="message">
+                <p>
+                    Hi there! Thank you for stopping by. I'm a software developer and love creating interesting things. If you have an idea that you need help with. I'll be happy to help bring it to life!
+                </p>
+            </div>
+        </div>
+        <div class="connect">
+            <div class="link">
+                <a href="#">Reach Out</a>
+            </div>
+            <div class="social">
+                <i class="fab fa-twitter"></i>
+                <i class="fab fa-instagram"></i>
+                <i class="fab fa-facebook"></i>
+                <i class="fab fa-linkedin"></i>
+            </div>
```

Comment:
- [ ] The list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (`<a>` tag). Please kindly fix that.